tket/src/Circuit/macro_circ_info.cpp

```diff
@@ -92,6 +92,30 @@ std::list<Command> Circuit::get_commands_of_type(OpType op_type) const {
   }
   return coms;
 }
+unsigned Circuit::count_n_qubit_gates(unsigned size) const {
+  unsigned counter = 0;
+  if (size == 0) return counter;
+  BGL_FORALL_VERTICES(v, dag, DAG) {
+    if (n_in_edges_of_type(v, EdgeType::Quantum) == size) {
+      const Op_ptr op_ptr = get_Op_ptr_from_Vertex(v);
+      switch (op_ptr->get_type()) {
+        case OpType::Input:
```

Comment:
Do you need ClInput and Cloutput here as well?

Reply:
I haven't added `CLInput` or `CLOutput` here as they should never have an in or out edge of `EdgeType::Quantum`.

Reply:
Ah, yes! Great!